templates/docs/examples/patterns/navigation/dropdown.html

```diff
@@ -7,9 +7,12 @@
 <header id="navigation" class="p-navigation">
   <div class="p-navigation__row">
     <div class="p-navigation__banner">
-      <div class="p-navigation__logo">
-        <a class="p-navigation__item" href="#">
-          <img class="p-navigation__image" src="https://assets.ubuntu.com/v1/ac9a0e00-lxd_primary.svg" alt="LXD" width="95" />
+      <div class="p-navigation__tagged-logo">
+        <a class="p-navigation__link" href="#">
+          <div class="p-navigation__logo-tag">
+            <img class="p-navigation__logo-icon" src="https://assets.ubuntu.com/v1/82818827-CoF_white.svg" alt="">
+          </div>
+          <span class="p-navigation__logo-title p-heading--4">LXD</span>
```

Comment:
This should not require additional `p-heading-4` class name, the `p-navigation__logo-title` class should provide all the necessary styling.

Heading styles can be included internally in SCSS we have placeholder or mixin for it.